docs/source/concepts/DynamicalStructureFactorFromAbInitio.rst

```diff
@@ -79,47 +85,50 @@ with
 :math:`\sigma_j` -- cross-section for :math:`j`-th atom
 
 
-The formula above is valid for *first order quantum events*, e. g., transitions :math:`0 \rightarrow 1` for each phonon. In order to
-reconstruct spectrum one has to also consider higher order quantum events. For second order quantum event one should
-consider transitions :math:`0 \rightarrow 2`, but also simultaneous transitions :math:`0 \rightarrow 1`, :math:`0 \rightarrow 1'` for different phonons. Within harmonic approximation all second
-order transitions form the following  set: :math:`\lbrace \omega_1 + \omega_1, \omega_1 + \omega_2, \omega_1 + \omega_3, \ldots,  \omega_p + \omega_p \rbrace`.
-Cardinality of this set is :math:`p^2`, where :math:`p` is a number of fundamentals. In practice one can reduce this number by taking into consideration realistic energy  window
+The formula above is valid for *first-order quantum events*, i.e. transitions :math:`0 \rightarrow 1` for each phonon. (We neglect events that transfer energy *to* the scattered neutron.)
+In order to reconstruct the full spectrum one has to also consider higher-order quantum events.
+For second-order quantum events one should
+consider transitions :math:`0 \rightarrow 2`, but also simultaneous transitions :math:`0 \rightarrow 1`, :math:`0 \rightarrow 1'` for different phonons.
+Within the harmonic approximation all second-order transitions form the following set: :math:`\lbrace \omega_1 +
+\omega_1, \omega_1 + \omega_2, \omega_1 + \omega_3, \ldots, \omega_p + \omega_p \rbrace`.
+The cardinality of this set is :math:`p^2`, where :math:`p` is a number of fundamentals.
+In practice one can reduce this number by taking into consideration a realistic energy window
```

Comment:
but if omega_j is large?

Reply:
I've been wondering the same thing. Still, this is documentation of how Abins works _at the moment_. I'd like to examine this approximation more in future...

Reply:
but it makes no sense due to the symmetry. If a is large and b is small, we are saying:
to use a+b
but that b+a is negligible. This will always be true as addition is commutative